### DISCUSSION TITLE
how to download data filtered by multi-filter?

### DISCUSSION THREAD

**Discussion by qq20230530** [getredash/redash#6120] [Q&A]
Noticed that when download CSV or Excel data, all data retrieved by queri is downloaded.
Is it possible to only download filtered data?

**Comment by gaecoli** [comment]
> Noticed that when download CSV or Excel data, all data retrieved by queri is downloaded. Is it possible to only download filtered data?

This is not currently supported and does not appear to be planned for the process.
